Update experiment user property limit (#614)
amplitude/amplitude-docs: content/collections/under-the-hood/en/event-tracking.md

## Experiment user properties

> Experiment uses a user property per flag and experiment, which is set or unset on both assignment and exposure events. Experiment uses this user property to determine which variant the user is in for experiment analysis purposes. You can have up to 1500 experiment user properties per project.

Reviewer: ```suggestion
Experiment uses a user property per flag and experiment, which is set or unset on both assignment and exposure events. Experiment uses this user property to determine which variant the user is in for experiment analysis purposes. Amplitude supports up to 1500 experiment user properties per project.
```
